src/skmultiflow/meta/leverage_bagging.py

```diff
@@ -265,16 +259,20 @@ def __partial_fit(self, X, y):
                 k = self._random_state.poisson(1)
```

Comment:
Sorry if the comment is misplaced (github doesn't let me comment on collapsed portions of the diff view). I think it would be nice to add short comments for each of the `if/elif/else` statements evaluating the values in `self.LEVERAGE_ALGORITHMS`. In this sense, we can easily track which Leveraging Bagging variant each portion of the code covers.

Reply:
Nice catch. Added comments in e979d59d7b7aee316c049a64d4540c07114848bc